contracts/single-collateral-factory/interfaces/IFactory.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: bsl-1.1
+
+/*
+  Copyright 2022 Unit Protocol: Artem Zakharov (az@unit.xyz).
+*/
+pragma solidity ^0.7.1;
+pragma experimental ABIEncoderV2;
+
+interface IFactory {
+
+    struct DeploySettings {
+        string stableName;
+        string stableSymbol;
+
+        address commonFeeCollector;
+        address issuanceFeeCollector;
+
+        address collateral;
+        address chainlinkAggregator;
+
+        uint16 issuanceFeeBasisPoints;
+        uint stabilityFeePercent3Decimals;
```

Comment:
Note that it's so dangerous to have a fixed `stabilityFeePercent` & fixed ratios - no money supply (and thus currency stability) regulation.